packages/ripple-nuxt-tide/lib/templates/plugin.js

```diff
@@ -93,9 +93,18 @@ export default ({ app, req, store , route }, inject) => {
         },
         async setSiteData ({ commit }, { requestId = null } = {}) {
           const headersConfig = { requestId }
-          const siteData = await app.$tide.getSiteData(headersConfig)
+          let siteData = await app.$tide.getSiteData(headersConfig)
           if (siteData instanceof Error) {
-            throw siteData
+            // If got a Nuxt auth session expire error, try again.
+            // As Nuxt auth should reset auth on error, retry should pass.
+            if (siteData.name === 'ExpiredAuthSessionError') {
+              siteData = await app.$tide.getSiteData(headersConfig)
```

Comment:
The extending of timeout didn't work? I feel like the main issue lies in the API somewhere and this is just to have a workaround.

Reply:
@rubyllamadora this is for a handle the auth token expiration error not about timeout, see https://github.com/dpc-sdp/ripple/pull/888#discussion_r562401166